packages/tasks/src/tasks/audio-classification/spec/input.json

```
@@ -0,0 +1,29 @@
+{
+	"$id": "/inference/schemas/audio-classification/input.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
```

```suggestion
	"$schema": "http://json-schema.org/draft-06/schema#",
        "title": "AudioClassificationInput",
```

(nit) missing `"title"` field in the input specs. How does quicktype know which name to give to the dataclass? (for output specs it seems `"title"` is there)

---

> How does quicktype know which name to give to the dataclass? 

I think it does some inference based on the `name` attribute we set here: https://github.com/huggingface/huggingface.js/blob/a33987fe219616a75c0e9fca79a65164472a9dd0/packages/tasks/src/scripts/inference-codegen.ts#L46

---

If possible I would explicitly add titles everywhere so that we stay consistent in case we get rid of quicktype at some point or if quicktype implicit naming rules change.

---

(in case it can help, I used this dirty script to update all of the schemas [in this PR](https://github.com/huggingface/inference-schemas/pull/4/files#diff-c02135df8f023a5f1631042fe89eb6454763fbeaa84698fa250a6912813be2db):

```py
from pathlib import Path

schemas_dir = Path("./schemas")

for schema_file in schemas_dir.glob("**/*.json"):
    task = schema_file.parent.name
    if "input" in str(schema_file):
        title = "".join(part.capitalize() for part in task.split("-")) + "Input"
    else:
        title = "".join(part.capitalize() for part in task.split("-")) + "Output"

    with open(schema_file, "r") as f:
        schema_lines = f.readlines()

    schema_lines.insert(4, f'    "title": "{title}",\n')
    with open(schema_file, "w") as f:
        f.writelines(schema_lines)
```
)

---

Done - thanks!